source/leuven/slurm_specifics.rst

```diff
@@ -0,0 +1,172 @@
+.. _leuven_slurm_specifics:
+
+Site-specific Slurm info
+========================
+
+While the :ref:`Running jobs in Slurm <running jobs>` pages provide generic
+information regarding Slurm, there are additional points to consider when
+when using Slurm on Tier-2 clusters hosted at KU Leuven.
```

Comment:
"when" is repeated

Reply:
👍 (238e439b1c908453cd3995ef47e2e67a9ed1a048)